### PR TITLE
feat: Add `getPronouns` method for mod integration

### PR DESCRIPTION
**Pull request by sleepingdragoninn**
Just adds a simple static `@Nullable Pronouns getPronouns(player)` method for other mods to use.

### CODE REVIEW

File: src/main/java/io/github/ashisbored/playerpronouns/PlayerPronouns.java [hunk 1]
```diff
@@ -191,4 +191,9 @@ public static boolean setPronouns(ServerPlayerEntity player, @Nullable Pronouns
 
         return true;
     }
+
+    public static @Nullable Pronouns getPronouns(ServerPlayerEntity player) {
```
Comment (ashhhleyyy) [inline src/main/java/io/github/ashisbored/playerpronouns/PlayerPronouns.java:196]: It's probably worth exposing a second version of this which takes a UUID directly too

File: build.gradle.kts [hunk 2]
```diff
@@ -5,7 +5,7 @@ plugins {
     `maven-publish`
 }
 
-version = "1.5.0+1.19"
+version = "1.5.1+1.19"
```
Comment (ashhhleyyy) [inline build.gradle.kts]: Nit: this version bump should be to `1.6.0` because its a new 'feature' rather than just a bugfix

**Review by ashhhleyyy — CHANGES_REQUESTED** [review]
I'll look at getting proper maven publishing setup for this, and I'll probably work on a full API interface for a `2.0.0` release at some point

**Review by sleepingdragoninn — COMMENTED** [review]
Sorted these two out. Let me know if there's anything else you'd like to tweak.

**Review by ashhhleyyy — APPROVED** [review]

**Pull request merged**